llama_index/embeddings/langchain.py

```diff
@@ -48,16 +49,31 @@ def class_name(cls) -> str:
         """Get class name."""
         return "LangchainEmbedding"
 
+    def _async_warn_once(self) -> None:
+        if not self._async_warned:
+            print("Async embedding not available, falling back to sync method.")
+            self._async_warned = True
+
     def _get_query_embedding(self, query: str) -> List[float]:
         """Get query embedding."""
         return self._langchain_embedding.embed_query(query)
 
     async def _aget_query_embedding(self, query: str) -> List[float]:
-        return await self._langchain_embedding.aembed_query(query)
+        try:
+            return await self._langchain_embedding.aembed_query(query)
+        except Exception:
```

Comment:
My only comment is we should catch a more specific exception -- `NotImplementedError` is probably the right one to catch

Reply:
@logan-markewich solved